cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java

```diff
@@ -72,28 +85,33 @@ public static void main(String[] args) {
         ApplicationContext context = new ClassPathXmlApplicationContext("spring/testRunnerContext.xml");
         SlangBuilder slangBuilder = context.getBean(SlangBuilder.class);
         Slang slang = context.getBean(Slang.class);
+        SlangTestCaseRunReportGeneratorService reportGeneratorService = context.getBean(SlangTestCaseRunReportGeneratorService.class);
         registerEventHandlers(slang);
 
+//        long time = System.currentTimeMillis();
         try {
-            SlangBuildResults buildResults = slangBuilder.buildSlangContent(projectPath, contentPath, testsPath, testSuites);
-            RunTestsResults runTestsResults = buildResults.getRunTestsResults();
+            SlangBuildResults buildResults = slangBuilder.buildSlangContent(projectPath, contentPath, testsPath, testSuites, runTestsInParallel);
+            IRunTestResults runTestsResults = buildResults.getRunTestsResults();
             Map<String, TestRun> skippedTests = runTestsResults.getSkippedTests();
 
-            if(MapUtils.isNotEmpty(skippedTests)){
+            if (isNotEmpty(skippedTests)) {
                 printSkippedTestsSummary(skippedTests);
             }
             printPassedTests(runTestsResults);
-            if(shouldPrintCoverageData) {
+            if (shouldPrintCoverageData) {
                 printTestCoverageData(runTestsResults);
             }
+//            System.out.println("Took " + (System.currentTimeMillis() - time) / 1000f + " s.");
 
-            if(MapUtils.isNotEmpty(runTestsResults.getFailedTests())) {
+            if (isNotEmpty(runTestsResults.getFailedTests())) {
                 printBuildFailureSummary(projectPath, runTestsResults);
-                System.exit(1);
             } else {
                 printBuildSuccessSummary(contentPath, buildResults, runTestsResults);
-                System.exit(0);
             }
+
+            reportGeneratorService.generateReport(runTestsResults, testsPath);
```

Comment:
I believe testsPath should be configurable from `cslang.properties` with a default value ~ `{app.home}/report`